patches/emacs-28/no-titlebar-and-round-corners.patch

```diff
@@ -23,3 +42,13 @@ index 1643ba5..4feca66 100644
    return self;
  }
  
+ 
+ - (void)createToolbar: (struct frame *)f
+ {
++  return;
+   if (FRAME_UNDECORATED (f) || !FRAME_EXTERNAL_TOOL_BAR (f))
```

Comment:
In general, `return` here is no not needed. But with the round corners patch, the frame is considered decorated, and so we need to make sure that any call to create toolbar noops.